client/filestore-client/src/CordovaFileSupport.ts

```diff
@@ -43,6 +44,11 @@ export class CordovaFileSupport {
           const blob = new Blob([new Uint8Array(this.result)], { type: 'image/jpg' });
           const data = new FormData();
           data.append('file', blob);
+          each(file, function(value, key) {
+            if (value) {
+              data.append(key, value);
+            }
+          });
```

Comment:
This just adds extra fields from the FileQueueEntry object to the request, currently the server ignores them, but I think it's good to send them too for future-proofing (TM)